packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -181,6 +186,15 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
             fluentTheme={fluentTheme}
             onClose={closePane}
             onFetchAvatarPersonaData={props.onFetchAvatarPersonaData}
+            onChatButtonClick={() => {
```

Comment:
nit: these need memoized

Reply:
Might be good to have a `paneState: 'closed' | 'chatOpen' | 'peopleOpen'` instead of two seperate showChat and showPeople that cannot both be true